src/axom/primal/geometry/Vector.hpp

```diff
@@ -63,6 +63,16 @@ template <typename T, int NDIMS>
 AXOM_HOST_DEVICE Point<T, NDIMS> operator+(const Vector<T, NDIMS>& V,
                                            const Point<T, NDIMS>& P);
 
+/*!
```

Comment:
These changes are related to https://github.com/LLNL/axom/issues/708
I think this completes the set.